crates/router_env/src/logger/types.rs

```diff
@@ -396,6 +396,8 @@ pub enum Flow {
     UpdateRole,
     /// User email flow start
     UserFromEmail,
+    /// Begin TOTP
+    TOTPBegin,
```

Comment:
Nit: naming convention.

```suggestion
    TotpBegin,
```